examples/db-templates/mongodb-persistent-template.json

```diff
@@ -169,6 +177,11 @@
   ],
   "parameters": [
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
+      "description": "Maximum amount of memory allowed by OpenShift Quota System",
```

Comment:
we should also start adding display names to parameters while we're in here wordsmithing